lib/api/_loaders/element-global.js

```diff
@@ -89,7 +90,23 @@ class ElementGlobal {
 
     const byLocator = Locator.create(locator);
 
-    this.element = await this.transport.driver.wait(until.elementLocated(byLocator), this.timeout, null, this.retryInterval);
+    this.setPropertiesFromElement(locator);
+
+    const elements = await this.transport.driver.wait(until.elementsLocated(byLocator), this.timeout, null, this.retryInterval);
```

Comment:
We can write the above as:

```js
if (locator instanceof Element) {
  this.setPropertiesFromElement(locator);

  locator = Locator.create(locator);
}

const elements = await this.transport.driver.wait(until.elementsLocated(locator), this.timeout, null, this.retryInterval);
```

You would need to change `const {locator}` to `let {locator}` for this to work.